### ISSUE TITLE
Reach weight limit while executing execute_xcm_transfer

### ISSUE THREAD

**Issue by clearloop** [ChainSafe/PINT#190]
## Issue summary

Testing `remote_asset` in #160 :

```
====> Error: assetIndex.removeAsset failed: remoteAssetManager.WeightLimitReached:  Execution of the XCM would potentially result in a greater weight used than the pre-specified, weight limit. The amount 
that is potentially required is the parameter.    
```

---

After giving enough WeightLimit

```
====> Error: assetIndex.removeAsset failed: remoteAssetManager.Barrier:  The XCM did not pass the barrier condition for execution. The barrier condition differs on different, chains and in different circumstances, but generally it means that the conditions surrounding the message, were not such that the chain considers the message worth spending time executing. Since most chains, lift the barrier to execution on appropriate payment, presentation of an NFT voucher, or based on the, message origin, it means that none of those were the case.
```

## Other information and links

-

**Comment by mattsse** [comment]
The current xcm weights needs some adjustments, this will hopefully be fixed with #161

**Comment by clearloop** [comment]
xToken errors after #161
